R/fb_aggregate_site_data.R

```diff
@@ -27,15 +27,15 @@
 #' @examples
 #' library("funbiogeo")
 #' 
-#' data("site_locs")
-#' data("species_occs")
+#' data("site_locations")
+#' data("site_species")
 #' 
 #' ## Import grid ----
 #' tavg <- system.file("extdata", "annual_mean_temp.tif", package = "funbiogeo")
 #' tavg <- terra::rast(tavg)
 #' 
 #' ## Rasterize 3 first species counts ----
-# FIXME (finish examples)
+# F# IXME (finish examples)
```

Comment:
Is there a reason for the change in FIXME?

Reply:
Nope. I will add the example now I have added new raster data.